src/components/Peers/GroupDialog.tsx

```diff
@@ -85,6 +84,9 @@ export const GroupDialog = ({ cur, save, remove, isAdmin, inUse }: IProps) => {
     setOpen(false);
   };
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const permissionTitles = React.useMemo(() => localizedPermissions(), []);
+
```

Comment:
this should have the dependency of permissions

Reply:
I made this change.